Formula/l/leiningen.rb

```diff
@@ -42,19 +44,19 @@ def caveats
   end
 
   test do
-    assert_equal version, resource("jar").version, "`jar` resource needs updating!"
```

Comment:
Doesn't this line already make sure resource is in sync?

Reply:
it would only check when the build is done, moving to install would be more preventive